konami/kpi.md

```diff
@@ -0,0 +1,190 @@
+# Konami Product Identifier (KPI)
+
+a.k.a. "mcodes"
+
+For identification purpose, Konami labels their products, e.g. full cabinet assembly, hardware +
+software upgrade kit, software update, using a propriatary format. These "product codes" are often
+seen denoting specific versions/iterations of software or hardware.
+
+## Sources
+
+* pigstall
```

Comment:
Better to cite individuals here, I think.

Reply:
That's rather difficult if the knowledge has been acquired by simply reading stuff there over many years.

Reply:
May be better to just replace it with a non-specific reference then -- I definitely know I got some of my knowledge on this from reading posts on various other websites.